Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -347,6 +348,38 @@ lemma gaussianReal_mul_const {X : Ω → ℝ} (hX : Measure.map X ℙ = gaussian
 
 end Transformations
 
+open Measurable Real
+
+variable {Ω : Type} [MeasureSpace Ω] {μ : ℝ} {v : ℝ≥0} {X : Ω → ℝ}
```

Comment:
There is a hierarchy of universes in Lean, and `Type` means a type in universe 1 (see here: https://leanprover.github.io/reference/expressions.html#universes). You should replace it with `Type*`, which means `Type u` for a newly created universe `u`: that makes the statements strictly more general with respect to universes.
Then don't use `MeasureSpace`, but use a `MeasurableSpace` and a measure. This is again for generality. A lemma written for any measure also applies to `volume` in a measure space, but the converse is not true. I know that other parts of this file use `MeasureSpace` and set a bad example, but we should fix that in another PR.
```suggestion
variable {Ω : Type*} {mΩ : MeasurableSpace Ω} {μ : ℝ} {v : ℝ≥0} {X : Ω → ℝ}
```